modules/serverless-functions-podman.adoc

```diff
@@ -7,21 +7,28 @@
 
 If you are using podman, you must run the following commands before getting started with {FunctionsProductName}:
 
-. Start the podman service that listens on port `1234`:
+. Start the podman service that serves the Docker API on a UNIX socket at `${XDG_RUNTIME_DIR}/podman/podman.sock`:
 +
 [source,terminal]
 ----
-$ podman system service --time=0 tcp:0.0.0.0:1234 & # let run in background or another terminal
+$ systemctl start --user podman.socket
 ----
 
+[NOTE]
+====
+On most systems, this socket is located at `/run/user/$(id -u)/podman/podman.sock`.
+====
+
```

Comment:
```suggestion
$ systemctl start --user podman.socket
----
+
[NOTE]
====
On most systems, this socket is located at `/run/user/$(id -u)/podman/podman.sock`.
====

```

One last small nit I saw in the preview, adding a `+` here will keep it part of one step and fix the numbering